### PR TITLE
Announce 2019 elections

### PR DESCRIPTION
**Pull request by binford2k**

### CODE REVIEW

File: _posts/2018-11-17-elections-2019.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Vox Pupuli has opened elections for the 2019 calendar year Project Management Committee.
+The elections were announced at Puppet's [Contributor Summit Online][cso].
+This blogpost will give you the context and how to become a candidate.
+
+> Vox Pupuli is a collective of Puppet module, tooling and documentation authors
+> all working together to ensure continued development on the code we maintain.
+
+(Source: [https://voxpupuli.org/](https://voxpupuli.org/))
+
+The Vox Pupuli project is now more than three years old. It represents many
```
Comment (binford2k) [inline _posts/2018-11-17-elections-2019.md]: I'm not actually sure what date is considered to be the genesis date of VP, so someone please correct this :)

Comment (bastelfreak) [reply _posts/2018-11-17-elections-2019.md]: I think nobody knows :D

File: _posts/2018-11-17-elections-2019.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+The PMC has set [Kris Bosland][kris] to be the Election Officer and to run the
+elections, as stated in the [Governance Document][gd]. Kris is not running
+for PMC re-election.
+
+Nominate yourself or someone else by following the directions [here][e].
+
+Important Dates:
+
+* Before *November 30 2018 23:59UTC*: Accepting nominations
+* Between *December 1 2019 12:00UTC* and *December 15 2018 23:59UTC*: Vote
```
Comment (bastelfreak) [inline _posts/2018-11-17-elections-2019.md]: depending on the amount of votes we get, I suggest extending this.  But this is probably something Kris can decide during December.

Comment (binford2k) [reply _posts/2018-11-17-elections-2019.md]: Yeah, I figured that two week periods (with another two weeks before terms take effect) would give us a bit of a buffer.

**Pull request merged**